### DISCUSSION TITLE
Is there a way to show / search by Metadata that is included in the pictures already?

### DISCUSSION THREAD

**Discussion by kladderadeng** [immich-app/immich#5611] [Q&A] [answered]
I have an existing, growing for over 2 decades, library I imported into Immich. And as far as I can see, Immich did a great job, especially with the face detection.

Reading up on the search function, it states that when using m: you can search for existing Metadata, but this does not seem to work for me.

Most of my library is tagged very detailed and has an extensive Tag tree that was maintained in the last decades with various programs like Lightroom and Digikam, to name just 2. All the tags are always saved in the pictures themselves.

Although I appreciate that AI search features of immich, searching for "a red car" is not enough for my purposes and I usually use my own tags, so I can specifically filter by all Red 1969 Chevrolet Chevelle Sedans for example, because my Tag Tree starts at Car --> Maker --> Model --> Year --> Colour.
When I try to search in Immich by m:chevrolet, it finds just nothing, although I have hundreds of images tagged with Chevrolet.

I am not sure if I am doing something wrong or if the m: feature only searches for keywords you see in the Info section when viewing a picture. This would mean that Immich currently ignores all existing Tags from my photos... where the next question would be if this is something that might get implemented in the future?

**Comment by bo0tzz** [comment, marked as answer]
Immich doesn't have tag support yet. See #2168 / #3518
